test/test_torch.py

```diff
@@ -8585,6 +8585,11 @@ def test_Size(self):
         self.assertEqual(len(x), 3)
         self.assertRaises(TypeError, lambda: torch.Size(torch.ones(3)))
 
+        self.assertIsInstance(x + (4, 5), torch.Size)
```

Comment:
We should check that these actually output the right thing !

Reply:
I cleaned up the whole test.